### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by fkrause98**
Add docs.
These are mostly HTML files generated by ex_docs.
You can check the docs being hosted here: https://lambdaclass.github.io/elixir_riak_core/readme.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,14 @@
...
-You can think of it as a building block for distributed and scalable systems.
+Riak Core is a building block for distributed and scalable systems
+in the form of an Erlang Framework.
+To learn more, you can check the useful links section in the repo linked above, and check the 
+[docs](https://lambdaclass.github.io/elixir_riak_core/readme.html) for more.
 If you want to set it up with Erlang, we also have an [up-to-date (OTP 25)
-tutorial](https://github.com/lambdaclass/riak_core_tutorial)
-# Riak Core:
+tutorial](https://github.com/lambdaclass/riak_core_tutorial). 
+# Riak Core
```
Comment (facundoolano) [inline README.md:10]:
```suggestion
## Riak Core
```
and update below

File: README.md [hunk 2]
```diff
@@ -1,32 +1,14 @@
...
-    - [Solution:](#solution)
-    - [Creating project.](#creating-project)
-    - [Limiting VM Memory.](#limiting-vm-memory)
-    - [Setting up csv.](#setting-up-csv)
-    - [Storing:](#storing)
-    - [Reading CSV:](#reading-csv)
-    - [Visualizing Results:](#visualizing-results)
-
-# Riax
 Riax is an Elixir wrapper for Riak Core.
```
Comment (facundoolano) [inline README.md:3]: Riak Core should be a link to the repo

File: README.md [hunk 3]
```diff
@@ -1,32 +1,14 @@
...
-    - [Reading CSV:](#reading-csv)
-    - [Visualizing Results:](#visualizing-results)
-
-# Riax
 Riax is an Elixir wrapper for Riak Core. 
-Riak Core is a distributed systems framework, written in Erlang.
-You can think of it as a building block for distributed and scalable systems.
+Riak Core is a building block for distributed and scalable systems
+in the form of an Erlang Framework.
+To learn more, you can check the useful links section in the repo linked above, and check the
```
Comment (facundoolano) [inline README.md:6]: "useful links" should be a link instead of mentioning "the repo linked above"

File: README.md [hunk 4]
```diff
@@ -1,32 +1,14 @@
...
-    - [Visualizing Results:](#visualizing-results)
-
-# Riax
 Riax is an Elixir wrapper for Riak Core. 
-Riak Core is a distributed systems framework, written in Erlang.
-You can think of it as a building block for distributed and scalable systems.
+Riak Core is a building block for distributed and scalable systems
+in the form of an Erlang Framework.
+To learn more, you can check the useful links section in the repo linked above, and check the 
+[docs](https://lambdaclass.github.io/elixir_riak_core/readme.html) for more.
```
Comment (facundoolano) [inline README.md:7]: instead of "the docs" you can say separately "the tutorial"  and "the API reference"

File: setup.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
+# Setup
```
Comment (facundoolano) [inline setup.md:1]: I think this setup is missing a very short snippet showing how the Riax API is used (to get the general idea, not all the details).

Also, I think that little snippet together with the "add riax as a dependency in mix" should be shown in the README. the conceptual background that you currently have in the README is ok but is less important than this.

File: README.md [hunk 6]
```diff
@@ -67,370 +49,40 @@ hash it and assign it to a Virtual Node. Riak fits really well here as it
...
 
-![The Ring](ring.png)
+![The Ring](https://raw.githubusercontent.com/lambdaclass/riak_core_tutorial/master/ring.png)
 
 Virtual Nodes share what is called a keyspace.
 The number of VNodes is fixed at cluster creation and a given hash value will
 always belong to the same partition (i.e. the same VNode). The VNodes in turn
 are evenly distributed across all available physical nodes. Note this
 distribution isn't fixed as the keyspace partitioning is: the VNode distribution
 can change if a physical node is added to the cluster or goes down.
```
Comment (facundoolano) [inline README.md:66]: you could end this conceptual section with a link to what to read next in the doc

File: README.md (hunk 2 above)
Comment (fkrause98) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (fkrause98) [reply README.md]: done!

File: README.md (hunk 4 above)
Comment (fkrause98) [reply README.md:8]: Done!

File: README.md (hunk 6 above)
Comment (fkrause98) [reply README.md:73]: I've added a brief text that points to the tutorial, what do you think about it?

File: setup.md (hunk 5 above)
Comment (fkrause98) [reply setup.md:1]: I've added some snippets in the readme on [this commit](https://github.com/lambdaclass/elixir_riak_core/pull/26/commits/3eb713232024fe0f55e619d7f015319ee02c178c) and an install section on [this commit](https://github.com/lambdaclass/elixir_riak_core/pull/26/commits/8bbd9b4048bca842e3d65e644397f81dce2e650f), wdy?

**Pull request merged**